src/main/java/internet/shop/controllers/cars/AddCarController.java

```diff
@@ -0,0 +1,49 @@
+package internet.shop.controllers.cars;
+
+import internet.shop.lib.Injector;
+import internet.shop.model.Car;
+import internet.shop.model.Manufacturer;
+import internet.shop.service.CarService;
+import internet.shop.service.ManufacturerService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("internet.shop");
+    private CarService carService = (CarService) injector
+            .getInstance(CarService.class);
+    private ManufacturerService manufacturerService = (ManufacturerService) injector
+            .getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/car/create/car/create_car.jsp")
+                .forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Manufacturer manufacturer = null;
+        try {
+            manufacturer = manufacturerService
+                    .get(Long.parseLong(req.getParameter("manufacturer_id")));
+        } catch (RuntimeException e) {
```

Comment:
it is not a good practice to catch RuntimeExceptions
let's remove try-catch here and assume that we will receive only valid ids here

Reply:
done